{
    cosvxx: 2.0,
    cosvyy: 2.0,
    cosxx: 0.7,
    cosyy: 0.7,
    measurement_noise: 1.3,
    overwrite_measurement_noise: true,
    process_noise_level: 0.34,
    use_process_noise: true
}
